database/seeds/ProjectsTableMassSeeder.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use App\Project;
+use Illuminate\Support\Facades\DB;
+
+/**
+ * Authors: Robert Breckenridge (original), Rick Boss (updater)
+ */
+class ProjectsTableMassSeeder extends MassSeeder
+{
+    private $images = '';
+
+    /**
+     * Initialize images path.
+     */
+    protected function setup()
+    {
+        $this->images = glob('public/images/seeds/*');
+    }
+    
+    /**
+     * Seeds with a randomly generated Project
+     *
+     * @param $iteration String: the current iteration
+     * @return mixed
+     */
+    public function seed($iteration)
+    {
+        factory(Project::class, 1)->create([
+            'author' => $this->faker->userName,
+            'title' => $this->faker->realText(32),
```

Comment:
The faker should return something else because sometimes it returns symbols that breaks the url.